docs/project-setup.md

```diff
@@ -0,0 +1,78 @@
+# Setting up a new project
+This document covers the basic project management tools and procedures recommended for Special Interest Group developing an implementation of the OpenTelemetry Specification for a given programming language.
+
+A Special Interest Group focused on implemeting OpenTelemetry in a programming language is reffered to as a **project** for the remainder of this document.
+
+If you are interested in starting a new project, please do the following
+* [create an issue](github.com/open-telemetry/community/issues) proposing the new project.
+* [Configure the new repository](docs/how-to-configure-new-repository.md).
+
+
+# Roadmap to Convergence
+For each new language, we want to quickly achieve parity with existing OpenTracing and OpenCensus implementations.
+
+[Merging OpenTracing and OpenCensus: A Roadmap to Convergence](https://medium.com/opentracing/a-roadmap-to-convergence-b074e5815289) defines the initial goals and timelines for OpenTelemetry.
+
+## OpenTelemetry Timeline
+**4/24** The reference candidate is presented for review.
+
+**5/8** Teams form, and work begins in all languages.
+
+**9/6** C#, Golang, Java, NodeJS, and Python reach parity with existing projects.
+
+**11/6** OpenTracing and OpenCensus projects are officially sunset. 
+
+## Initial Milestones
```

Comment:
Please review the following milestones.
* Are these the correct list of milestones?
* I left the details for each milestone vague, but I would like to fill them in as a group.

Reply:
My first thought is that may be we should consider having the milestones/roadmap/dates in only one place say website. The more places we have the roadmap the more places we need to track and update as and when we change milestones, dates etc. (Assumption here is that this doc would be included in the repo for every project. if incorrect you can ignore above)

Also with regards to the milestones and dates I propose we use what we have https://docs.google.com/document/d/1FKBlwrsjKMIHuyxzjmqPRJNnqhvZ-Ftt1K1cv8NPPxQ/edit?ts=5cdc9754#heading=h.oh8xyon7gukv instead of hard dates.

Reply:
### Q: What dates should we use?
I'm fine with the looser dates. Or alternatively, we could go with relative dates, since there may be some projects which start later?

### Q: Don't we also want all of this content on the website?
I believe this issue will come up a lot. 

On the one hand, we don't want to update a separate document, just to put content on the website. On the other hand, different content has different review process, with different community members. For example, specification changes are different than ruby documentation changes. So there is a puzzle.

One solution is to use use submodules, and embed pages from the community and specification repos directly in the website. Another solution would be to get rid of the community and specification repos, and make the website repo canonical. Seems like an issue worthy of it's own issue. :)

Reply:
BTW I ended removing most of the timeline references, and used the looser dates of "September" and "November" when they do appear.

I think the question of how to replicate community and spec content on the website should be brought up in a separate issue on the website repo. I'll make one today.